### ISSUE TITLE
Inconsistency in `pupil.py`

### ISSUE THREAD

**Issue by GillesOrban** [vortex-exoplanet/HEEPS#27]
This change has apparently disappeared...

_Originally posted by @GillesOrban in https://github.com/vortex-exoplanet/HEEPS/issues/19#issuecomment-616501402_

**Comment by GillesOrban** [comment]
This dic:
```
    pupil_file = {
    0: conf['pupil_file'],
    1: 'ELT_2048_37m_11m_5mas_nospiders_1missing_cut.fits',
    2: 'ELT_2048_37m_11m_5mas_nospiders_2missing_cut.fits',
    4: 'ELT_2048_37m_11m_5mas_nospiders_4missing_cut.fits',
    7: 'ELT_allglass_flower_253.fits'}
```
should not be hardcoded in `pupil.py` but in `read_config.py`, if needed.

**Comment by ChrisDelaX** [comment]
Fixed by e613bc33f4227e44b177ed53bf9fbbc28fd0c5b4
